core/src/main/resources/db/migration/V28__add_campaign_created_date.sql

```diff
@@ -0,0 +1,13 @@
+ALTER TABLE `Campaign`
+ADD COLUMN created_at DATETIME(3) NOT NULL DEFAULT NOW();
+
+ALTER TABLE `Package`
+ADD COLUMN created_at_temp DATETIME(3) NOT NULL DEFAULT NOW();
+
+UPDATE `Package`
+SET created_at_temp = createdAt;
+
+ALTER TABLE `Package`
+DROP COLUMN createdAt;
+ALTER TABLE `Package`
+CHANGE created_at_temp created_at DATETIME(3) NOT NULL DEFAULT NOW();
```

Comment:
Ah, I didn't know about temporary tables. I'll remember this for the future, thanks!